.gitignore

```
@@ -22,6 +22,8 @@ cabal.config
 *.swp
 *.swo
 *#
+*yaml.lock
```

Why not just *.lock?

```suggestion
*.lock
```